src/main/java/com/entropyteam/entropay/employees/projects/models/Project.java

```diff
@@ -0,0 +1,44 @@
+package com.entropy.entropay.employees.projects.models;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.UUID;
+import com.entropy.entropay.employees.clients.models.Client;
+
+import lombok.Data;
+
+@Entity
+@Table(name = "projects")
```

Comment:
some tables have the schema specified and other no.

Reply:
Project entity wasn't part of this pr